spec.bs

```diff
@@ -541,54 +570,64 @@ On the other hand, methods for getting data from the [=shared storage database=]
     The <dfn method for="WindowSharedStorage">set(|key|, |value|, |options|)</dfn> method steps are:
 
     1. Let |promise| be a new [=promise=].
-    1. Let |context| be {{WindowSharedStorage}}'s {{Window}}'s [=Window/browsing context=].
-    1. [=Assert=] that |context| is not null.
     1. If |key|'s [=string/length=] exceeds the [=key/maximum length=], return a [=promise rejected=] with a {{TypeError}}.
     1. If |value|'s [=string/length=] exceeds the [=value/maximum length=], return a [=promise rejected=] with a {{TypeError}}.
+    1. Let |context| be {{WindowSharedStorage}}'s {{Window}}'s [=Window/browsing context=].
```

Comment:
Can this instead be `[=this=]'s [=relevant global object=]'s [=Window/browsing context=]` (like in https://wicg.github.io/fenced-frame/#dom-fence-reportevent)? Just reaching for `{{WindowSharedStorage}}` object I think it less good than using `[=this=]`.

this comment applies to the instances just like this below too (there seem to be at least 3 maybe)